extractor/src/main/java/org/schabi/newpipe/extractor/services/youtube/YoutubeParsingHelper.java

```diff
@@ -108,7 +108,12 @@ public static boolean isInvidioURL(URL url) {
         return host.equalsIgnoreCase("invidio.us") || host.equalsIgnoreCase("dev.invidio.us") || host.equalsIgnoreCase("www.invidio.us") || host.equalsIgnoreCase("invidious.snopyta.org") || host.equalsIgnoreCase("de.invidious.snopyta.org") || host.equalsIgnoreCase("fi.invidious.snopyta.org") || host.equalsIgnoreCase("vid.wxzm.sx") || host.equalsIgnoreCase("invidious.kabi.tk") || host.equalsIgnoreCase("invidiou.sh") || host.equalsIgnoreCase("www.invidiou.sh") || host.equalsIgnoreCase("no.invidiou.sh") || host.equalsIgnoreCase("invidious.enkirton.net") || host.equalsIgnoreCase("tube.poal.co") || host.equalsIgnoreCase("invidious.13ad.de") || host.equalsIgnoreCase("yt.elukerio.org");
     }
 
-    public static long parseDurationString(String input)
+    /**
+     * Parses the duration string of the video expecting ":" or "." as seperators
+     * @return the duration in seconds
+     * @throws ParsingException when more than 3 seperators are found
+     */
+    public static int parseDurationString(String input)
```

Comment:
```suggestion
    public static int parseDurationString(final String input)
```

Reply:
`final` at this place is actually enforced by our Checkstyle rules, though obviously I haven't made the PR for using Checkstyle on NewPipeExtractor yet.